### ISSUE TITLE
plz setup fails on fresh clone

### ISSUE THREAD

**Issue by hackedhead** [m3brown/plz#1]
I believe this is due to the repo not having a Pipfile for pipenv?

```
[14:47] hackedhead@buzzy:~/projects % cd plz 
[14:47] hackedhead@buzzy:~/projects/plz % plz setup

[INFO] Using config: .plz.yaml

===============================================================================
Running command: pipenv install --skip-lock -r requirements-dev.txt
===============================================================================

[pipenv.exceptions.PipfileNotFound]:   File "/usr/lib64/python2.7/site-packages/pipenv/vendor/click/decorators.py", line 64, in new_func
[pipenv.exceptions.PipfileNotFound]:       return ctx.invoke(f, obj, *args, **kwargs)
[pipenv.exceptions.PipfileNotFound]:   File "/usr/lib64/python2.7/site-packages/pipenv/vendor/click/core.py", line 555, in invoke
[pipenv.exceptions.PipfileNotFound]:       return callback(*args, **kwargs)
[pipenv.exceptions.PipfileNotFound]:   File "/usr/lib64/python2.7/site-packages/pipenv/vendor/click/decorators.py", line 17, in new_func
[pipenv.exceptions.PipfileNotFound]:       return f(get_current_context(), *args, **kwargs)
[pipenv.exceptions.PipfileNotFound]:   File "/usr/lib64/python2.7/site-packages/pipenv/cli/command.py", line 249, in install
[pipenv.exceptions.PipfileNotFound]:       editable_packages=state.installstate.editables,
[pipenv.exceptions.PipfileNotFound]:   File "/usr/lib64/python2.7/site-packages/pipenv/core.py", line 1713, in do_install
[pipenv.exceptions.PipfileNotFound]:       raise exceptions.PipfileNotFound(project.path_to("Pipfile"))
ERROR: /home/hackedhead/projects/plz/Pipfile not found! Aborting! Please ensure that the file exists and is located in your project root directory.

[ERROR] Process failed

===============================================================================
Skipping command due to previous errors: 'pipenv run pre-commit install'
===============================================================================
```

**Comment by m3brown** [comment]
Confirmed, this is a problem introduced by a more recent version of pipenv. It doesn't happen with, for example, pipenv==11.9.0 from March 2018. Thanks for reporting, I'll get this fixed.

**Comment by m3brown** [comment]
FWIW, for transparency: I was installing via requirements-dev.txt due to issues getting Travis to support pipenv for unit tests.  After some quick googling, it looks like there are some emerging patterns for running pipenv in Travis I need to check out.

**Comment by hackedhead** [comment]
NB: running `pipenv --python 3` to generate a Pipfile, and then continuing on to `plz setup` appears to work fine.

**Comment by m3brown** [comment]
@hackedhead I *think* I fixed this in the 0.5.1 release. If you have time, I'd appreciate it if you could give it a whirl.

**Comment by hackedhead** [comment]
This works for me now. It complains appropriately if `pipenv` is not available, but that's beyond the scope of this.
